spec/models/orphan_spec.rb

```diff
@@ -97,13 +94,13 @@
     end
 
     context 'when father is not alive' do
+      it { is_expected.to have_validation :valid_date_presence, :on => :father_date_of_death }
+      it { is_expected.to have_validation :date_not_in_future, :on => :father_date_of_death }
+
       specify 'father_date_of_death must be present and should be in the past' do
         orphan.father_alive = false
-        expect(orphan).to validate_presence_of :father_date_of_death
         expect(orphan).to allow_value(5.days.ago, Date.current).for :father_date_of_death
-        [7, 'yes', true, 5.days.from_now].each do |bad_date_value|
-          expect(orphan).to_not allow_value(bad_date_value).for :father_date_of_death
-        end
+        expect(orphan).to_not allow_value(5.days.from_now).for :father_date_of_death
```

Comment:
I think have it doesn't cause so much noise. Anyways, you are testing the wrong path, that I consider could be useful in this case.